web/src/components/troubleshoot/GenerateSupportBundle.jsx

```diff
@@ -347,11 +282,16 @@ class GenerateSupportBundle extends React.Component {
           return;
         }
         const response = await res.json();
+        this.props.updateBundleSlug(response.slug);
         this.setState({ newBundleSlug: response.slug });
-        this.state.pollForBundleAnalysisProgress.start(
-          this.pollForBundleAnalysisProgress,
-          1000
+
+        this.props.history.push(
+          `/app/${watch.slug}/troubleshoot/analyze/${response.slug}`
```

Comment:
redirects to the support bundle which is `SupportBundleAnalysis` component, there the progress bar will show.